src/App.js

```diff
@@ -181,23 +191,63 @@ const App = () => {
 		tools.push({name: name, icon: icon})
 	}
 
+	const login = () => {
+		fetch("https://skilling-59c90.firebaseio.com/anthony.json")
+      .then(res => res.json())
+      .then(
+        (result) => {
+					updateLevel("wood", woodcutting, setWoodcutting, result.woodXp)
+					updateLevel("stone", quarrying, setQuarrying, result.stoneXp)
+					updateLevel("fish", fishing, setFishing, result.fishingXp)
+					updateLevel("plank", makePlank, setMakePlank, result.plankXp)
+					updateLevel("fire", makeCampfire, setMakeCampfire, result.fireXp)
+					updateLevel("cook", cooking, setCooking, result.cookXp)
+					updateLevel("prospect", prospecting, setProspecting, result.prospectXp)
+					updateLevel("tool", toolMaking, setToolMaking, result.toolXp)
+					updateLevel("sacrifice", sacrifice, setSacrifice, result.sacrificeXp)
+        },
+        (error) => {
+          this.setState({
+            isLoaded: true,
+            error
+          });
+        }
+      )
+	}
+	const save = () => {
+		const userData = {woodXp: woodcutting.exp,
+											stoneXp: quarrying.exp,
+											fishingXp: fishing.exp,
+											plankXp: makePlank.exp,
+											fireXp: makeCampfire.exp,
+											cookXp: cooking.exp,
+											prospectXp: prospecting.exp,
+											toolXp: toolMaking.exp,
+											sacrificeXp: sacrifice.exp}
+		const response = fetch("https://skilling-59c90.firebaseio.com/anthony.json", {
+	    method: 'PUT',
+	    body: JSON.stringify(userData)
+	  });
+	}
+
 	useEffect(() => {
 		//Unlock Tool: Saw
-		if (woodcutting.exp === 9 && tools.some(e => e.name !== 'hammer')) {
```

Comment:
This wasn't working as you intended

In tools, is there some/any value where name != hammer? (that's nearly always true)